docs/slurm-cluster.md

```diff
@@ -54,11 +54,26 @@ Instructions for deploying a GPU cluster with Slurm
    ansible-playbook -l slurm-cluster playbooks/slurm-cluster.yml
    ```
 
+6. Verify Pyxis and Enroot can run GPU jobs across all nodes.
+
+  ```sh
+  # NOTE: This will use Pyxis to download a container and verify GPU functionality across all compute nodes 
+  ansible-playbook -l slurm-cluster playbooks/slurm-validation.yml -e '{num_gpus: 1}'
+  ```
 ## Using Slurm
 
 Now that Slurm is installed, try a ["Hello World" example using MPI](../examples/slurm-mpi-hello/README.md).
 
 
+## Monitoring Slurm
+
+As part of the Slurm installation, Grafana and Prometheus are both deployed.
+
+The services can be reached from the following addresses:
+* Grafana: http://mgmt:3000
```

Comment:
Instead of `http://mgmt:3000`, can you restate it to make it clear that `mgmt` is the Slurm controller node? This looks like it should be obvious, but I have seen this kind of description confuse people before. :( 

Either by replacing the URLs directly with something like `http://<slurm-controller>:3000`, or by adding a sentence following the bullets like "where `mgmt` is the name of the Slurm controller node".

Reply:
Yeah I agree that it needs clarity. I was just copy/pasting from the k8s doc for now. I think both of them should be updated here a bit. I would probably want to call it out using the groups in the inventory file though.

Reply:
Okay, I made this change and also updated all the other docs. the IPs now reference http://<slurm-master>:PORT and  http://<kube-master>:PORT univerally. This should make it more clear that the IP or hostname matches what is in the inventory file (as long as they haven't set up some sort of loadbalancer to hit their mgmt services).

Thoughts?